.stylelintignore

```diff
@@ -1,2 +1 @@
-packages/application-components/
-website/
\ No newline at end of file
+website/
```

Comment:
`stylelint` has an option `allow-empty-input` which if not passed makes stylelint fail if pointed at a location which is ignored again.